hortum_mobile/analysis_options.yaml

```diff
@@ -0,0 +1,29 @@
+include: package:flint/analysis_options.dart.dev.yaml
+#include: package:pedantic/analysis_options.1.8.0.yaml
+# include: package:lint/analysis_options.yaml
```

Comment:
Precisa desses comentários mesmo ou so o flint que vai ser utilizado ? Se posteriormente for útil para vocês ai não tem problema, mas se não forem usar acredito que pode apagar mesmo